MSET9_installer_script/MSET9-Windows.bat

```diff
@@ -1,12 +1,24 @@
 @echo off
 chcp 65001 > nul
-py -V > nul
-if %errorlevel% NEQ 0 (
-    echo Python 3 is not installed.
-    echo Please install Python 3 and try again.
-    echo https://www.python.org/downloads/
-    echo.
-    pause
-    exit
+
+set PY1="py"
+set PY2="%WINDIR%\py"
+set PY3="%LOCALAPPDATA%\Programs\Python\Launcher\py"
+
+setlocal enableDelayedExpansion
+for /l %%x in (1, 1, 3) do (
```

Comment:
`(1, 2 3)`?

Reply:
nope, that's stepping syntax for batch, it means stepping from 1, with 1 increment per step, upto 3